### ISSUE TITLE
Add translations for `IndianList` and `lang:hi`

### ISSUE THREAD

**Issue by slavaleleka** [AdguardTeam/FiltersRegistry#610]
due to:
https://github.com/AdguardTeam/FiltersRegistry/pull/609/

also filter description might be improved

**Comment by DandelionSprout** [comment]
I've now got confirmation that Nepal's main languages (Not just Nepali, but also Maithili) are also supported, even if it has yet to be added to their Readme: https://github.com/mediumkreation/IndianList/issues/5063#issuecomment-1109982902

So there'd be grounds to specify "Websites in Indian, Sri Lankan, and Nepali languages" in AdGuard's description string for the list.

**Comment by zloyden** [comment]
Yep, but we already included it in [another context](https://github.com/AdguardTeam/FiltersRegistry/pull/611/files).
